### DISCUSSION TITLE
Response code of 200 and LIBCURL ERROR: Failure writing output to destination mean in libcurl C++

### DISCUSSION THREAD

**Discussion by sayoojbk** [curl/curl#8568] [Q&A] [answered]
I am trying to connect to a SSL secured website and send POST through libcurl C++ https://curl.se/libcurl/c/ I see it shows SSL certificates verified OK but gives the output as Response code 200 with LIBCURL ERROR: Failure writing output to destination...

**Comment by bagder** [comment, marked as answer]
Check your write callback (set with `CURLOPT_WRITEFUNCTON`) and in particular make sure it returns the correct return code.

**Reply by sayoojbk** [reply]
Thanks a lot that solved my issue. Not related to this issue but wanted to know which files does libcurl take for HTTPS request with certificates and key. I passed the certification.pem file used in linux for our windows compilation but it shows Response 0. Any idea how I can solve this

**Reply by bagder** [reply]
It will be much better if you start a new "discussion" when you have an entirely separate question.
